manager/src/main/java/com/redhat/service/smartevents/manager/api/user/BridgesAPI.java

```diff
@@ -146,4 +149,21 @@ public Response deleteBridge(@PathParam("bridgeId") String bridgeId) {
         bridgesService.deleteBridge(bridgeId, identityResolver.resolve(jwt));
         return Response.accepted().build();
     }
+
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = BridgeListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Unauthorized.", responseCode = "401", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Forbidden.", responseCode = "403", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Not found.", responseCode = "404", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Internal error.", responseCode = "500", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class)))
+    })
+    @Operation(summary = "Get the list of Bridge instances", description = "Get the list of Bridge instances for the authenticated user.")
+    @GET
+    @Path("{bridgeId}/errors")
+    public Response getBridgeErrors(@PathParam("bridgeId") String bridgeId, @Valid @BeanParam QueryResourceInfo queryInfo) {
```

Comment:
@kostola I also wonder if we want to create another `API` class here, specifically called something like `EndpointErrorHandlerAPI`, again for the reasons of keeping the implementation as a kind of "plugin" rather than a core part of the Bridge API.

WDYT?

Reply:
Moved to another API class and added also to the OpenAPI YAML :+1: